### DISCUSSION TITLE
Registering new blocks

### DISCUSSION THREAD

**Discussion by beyond-these-walls** [jeffreyvr/dropblockeditor#26] [Q&A]
Tried making a new block:
php artisan dropblockeditor:make Text

Correct files are created:
BLOCK PATH: /var/www/html/app/DropBlockEditor/Blocks/Text.php
EDIT COMPONENT PATH: /var/www/html/app/Livewire/Text.php

Added the following to the config file:
return [     
   'blocks' => [         
      Jeffreyvr\DropBlockEditor\Blocks\Test::class,    
    ], 
]

Got error:
Class "Jeffreyvr\DropBlockEditor\Blocks\Test" not found

So tried the following:

App\DropBlockEditor\Blocks\Test::class,   
\App\DropBlockEditor\Blocks\Test::class,   

Got error:
Class "App\DropBlockEditor\Blocks\Test" not found

How can I register a new block?

**Comment by jeffreyvr** [comment]
Your Block is called `Text` but your registering a block with the name `Test` :)
